risc0/zkvm/src/host/client/prove/mod.rs

```diff
@@ -163,6 +167,8 @@ pub struct ProverOpts {
     /// programs that are allowed to run and is a key field in the
     /// [SuccinctReceiptVerifierParameters][crate::SuccinctReceiptVerifierParameters].
     pub control_ids: Vec<Digest>,
+    /// Maximum cycle count, as a power of two (po2) that these prover options support.
+    pub(crate) segment_limit_po2: usize,
```

Comment:
I think of this more as the `max_po2` rather than directly associated with the `segment_limit_po2`. Like `segment_limit_po2` on the `ExecutorEnv` can be smaller than `max_po2`, they seem distinct to me.

Reply:
Addressed in cc0c9dce